core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-page-tools-seo/dot-page-tools-seo.component.html

```diff
@@ -0,0 +1,33 @@
+<p-dialog
+    class="page-tools-dialog"
+    [(visible)]="visible"
+    [position]="'right'"
+    [closable]="false"
+    [draggable]="false"
+    header="{{ 'editpage.toolbar.nav.page.tools' | dm }}"
+>
+    <div class="page-tools">
+        <ul class="page-tools-list">
+            <li class="page-tools-list__item" *ngFor="let tool of tools$ | async">
+                <a class="page-tools-list__link" href="{{ tool.runnableLink }}" target="_blank">
+                    <img
+                        src="{{ tool.icon }}"
+                        alt="{{ tool.title }}"
+                        style="width: 2.6rem; height: 2.6rem"
+                    />
+                    <div class="page-tools-list__content">
+                        <span class="page-tools-list__title">{{ tool.title }}</span>
+                        <p class="page-tools-list__description">
+                            {{ tool.description }}
+                        </p>
+                        <div class="page-tools-list__tags">
+                            <div class="page-tools-list__tag" *ngFor="let tag of tool.tags">
+                                {{ tag }}
+                            </div>
```

Comment:
Can you check with Melissa if this is not the chip component? https://primeng.org/chip